sites/all/themes/vacation/scripts/carbon_copy.js

```diff
@@ -67,5 +67,57 @@
       removeCCUid($userItem.data('uid'));
       $userItem.remove();
     });
+
+    if ($('.page-vacation-request').length) {
```

Comment:
@daisy23333 Better to use another selector here, this is URL alias related. Can we use the form related selector?

Reply:
@jieyyal  Changed to form class "node-vocation-form" and move the function to a new js "vacation_autocomplete.js", please check.